_episodes/08-more-features-of-glatos.md

```diff
@@ -8,21 +8,41 @@ questions:
 
 glatos has some more advanced analytic tools beyond filtering and creating events.
 
+
```

Comment:
title and questions need to be changed to glatos

Reply:
Fixed